### PR TITLE
RD-1833 Update filters docs

### PR DESCRIPTION
**Pull request by YoniItzhak**
Update the filters REST docs to reflect the new complex filters mechanism.

### CODE REVIEW

File: content/_filters.md [hunk 1]
```diff
@@ -2,34 +2,57 @@
...
 `id` | string | The filter's ID, unique per tenant.
-`value` | dict | The filter's rules. Currently, of the form {'labels': {'not_equal': {}, 'equal': {}, 'is_null': [], 'is_not_null': []}}
-`labels_filters` | list | A list of the labels filters as strings.
+`value` | list | A list of the filter's filter-rules.
+`labels_filter_rules` | list | A list of the filter's filter-rules of type `label`.
+`attrs_filter_rules` | list | A list of the filter's filter-rules of type `attribute`.
 `visibility` | string | Defines who can see the filter. Can be private, tenant or global.
 `created_at` | datetime | The time when the filter was created.
 `updated_at` | datetime | The time the filter was last updated at.
+`tenant_name` | string | he name of the tenant that owns the filter.
```
Comment (glukhman) [inline content/_filters.md:55]:
```suggestion
`tenant_name` | string | The name of the tenant that owns the filter.
```

File: content/_filters.md [hunk 2]
```diff
@@ -2,34 +2,57 @@
...
-
-* `x=[y,z]`: All objects with the label `x:y or x:z`.
-
-* `x!=y`: All objects with the label `x:<any value other than y>`.  
-
-* `x!=[y,z]`: All objects with the label `x:<any value other than y and z>`.
+`<AttrsOperator>` can be one of: "any_of", "not_any_of", "contains", "not_contains", "starts_with", "ends_with", "is_not_empty".
 
-* `x is null`: All objects that don’t have the label `x:<any value>`.
+`<FilterRuleType>` can be one of: "label" or "attribute". If "label" is provided, then the operator must be a `<LabelsOperator>`, and if "attribute" is provided, then
```
Comment (glukhman) [inline content/_filters.md:19]:
```suggestion
`<FilterRuleType>` can be either "label" or "attribute". If "label" is provided, then the operator must be a `<LabelsOperator>`, and if "attribute" is provided, then 
```

File: content/_searches.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+}
+```
+`<LabelsOperator>` can be one of: "any_of", "not_any_of", "is_null" or "is_not_null".
+
+`<AttrsOperator>` can be one of: "any_of", "not_any_of", "contains", "not_contains", "starts_with", "ends_with", "is_not_empty".
+
+`<FilterRuleType>` can be one of: "label" or "attribute". If "label" is provided, then the operator must be a `<LabelsOperator>`, and if "attribute" is provided, then 
+the operator must be an `<AttrsOperator>`. 
+
+E.g. filtering by the following filter rules, will return all items of a resource that their creator name starts with "alice" or "bob",
```
Comment (glukhman) [inline content/_searches.md:25]:
```suggestion
E.g. filtering by the following filter rules, will return all items of a resource whose creator name starts with "alice" or "bob", 
```

File: content/_filters.md (hunk 2 above)
Comment (YoniItzhak) [reply content/_filters.md:19]: Changed.

File: content/_filters.md (hunk 1 above)
Comment (YoniItzhak) [reply content/_filters.md:55]: Changed.

File: content/_searches.md (hunk 3 above)
Comment (YoniItzhak) [reply content/_searches.md:25]: Changed here and in _filters.md

**Review by glukhman — APPROVED** [review]

**Pull request merged**